cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -195,15 +195,15 @@ private boolean initAdminClient() {
 
         Promise<Void> result = Promise.promise();
         singleExecutor.submit(() -> {
-            List<Integer> podIds = new ArrayList<>(numPods);
+            List<Integer> podIds = new ArrayList<>(podList.size());
```

Comment:
I do not think they are podIds anymore => they are now just indexes in the list. So I think it should be names appropriately here and in th other places (in the for loop etc.).